apps/web-giddh/src/app/shared/aside-menu-create-tax/aside-menu-create-tax.component.ts

```diff
@@ -57,15 +61,22 @@ export class AsideMenuCreateTaxComponent implements OnInit, OnChanges, OnDestroy
     public taxAuthorityList: IOption[] = [];
     /** Holds true if tax authority list is inprogress */
     public isTaxAuthoritiesLoading$: Observable<any> = this.componentStore.isLoading$;
+    /** Holds tax form */
+    public taxForm: FormGroup;
+    /** Holds if UntypedFormArray is valid or not */
+    public isValidForm: boolean = true;
+    /** Holds TDS TCS tax sub type */
+    private subTyp: string = '';
```

Comment:
```suggestion
    private subType: string = '';
```Please change in over all file